File: TCDNew-master/src/ui.wince/Cen.Wms.Client/Models/Dtos/PurchaseTaskDto.cs
using System;

namespace Cen.Wms.Client.Models.Dtos
{
    public class PurchaseTaskDto
    {
        public string Id { get; set; }
        public string Code { get; set; }
        public DateTime CreatedAt { get; set; }
        public DateTime ChangedAt { get; set; }

        public string DisplayText
        {
            get { return CreatedAt.ToLocalTime().ToString("yyyy.MM.dd HH:mm") + ", " + Code; }
        }
    }
}

File: TCDNew-master/src/ui.android/lib/models/dtos/PurchaseTaskDto.dart
class PurchaseTaskDto {
  String id;
  String code;
  DateTime createdAt;
  DateTime changedAt;

  PurchaseTaskDto(this.id, this.code, this.createdAt, this.changedAt);
  factory PurchaseTaskDto.fromJson(Map<String, dynamic> json) {
    PurchaseTaskDto data;

    if (json != null) {
      data = PurchaseTaskDto(
          json['id'],
          json['code'],
          DateTime.fromMillisecondsSinceEpoch(json['createdAt'], isUtc: true),
          DateTime.fromMillisecondsSinceEpoch(json['changedAt'], isUtc: true)
      );
    }

    return data;
  }
}